cogl/cogl/cogl-texture.c

```diff
@@ -1162,19 +1162,8 @@ cogl_texture_get_data (CoglTexture *texture,
    * texture and reading the pixels from the framebuffer. */
```

Comment:
Would you mind updating the comment as well, so that it's consistent with what's done below (no fallback from now on), ideally using the same one set in https://git.gnome.org/browse/cogl/commit/cogl/cogl-texture.c?id=6d6a277b8e9a63a8268046e5258877ba94a1da5b?

Reply:
Never mind, I "fixed" that and pushed already: https://github.com/endlessm/mutter/commit/4677635d2c140f0e89ad540ae5e473c49f6f03c4

Reply:
Oh, strange. I thought I had changed the comment too... at least, that was my intention

Reply:
Not a problem at all :)